frontend/src/Context/AudioOutputProvider/useAudioOutput/useAudioOutput.tsx

```diff
@@ -1,11 +1,14 @@
+import {
+  getActiveAudioOutputDevice as getVonageActiveAudioOutputDevice,
+  setAudioOutputDevice as setVonageAudioOutputDevice,
+} from '@vonage/client-sdk-video';
 import { useCallback, useEffect, useState } from 'react';
-import { getActiveAudioOutputDevice, setAudioOutputDevice } from '@vonage/client-sdk-video';
 
 export type AudioDeviceId = string | null | undefined;
 
 export type AudioOutputContextType = {
-  audioOutput: string | undefined | null;
-  setAudioOutput: (deviceId: AudioDeviceId) => Promise<void>;
+  currentAudioOutputDevice: string | undefined | null;
+  setAudioOutputDevice: (deviceId: AudioDeviceId) => Promise<void>;
```

Comment:
Some light renaming/refactoring here. It was super confusing to know what `updateAudioOutput ` `setAudioOutput `, `setAudioOutputDevice` etc all did and how they differed